templates/digital-asset-template/.gitignore

```diff
@@ -30,3 +30,6 @@ pnpm-lock.yaml
 *.njsproj
 *.sln
 *.sw?
+
+# template related files
+example-collection/
```

Comment:
maybe we include this in git as a reference?

Reply:
you mean instead of having it in code?

Reply:
I mean don't ignore it, just keep it like current status quo

Reply:
we add it to the generated template's gitignore, i.e what the dev will have when generating a template. so if they push it to their own github repo, it would be ignored.

Reply:
ah i see, nice!